docs/README.md

```diff
@@ -124,7 +124,7 @@ Events are the cornerstone of a Booster application, and that’s why we say tha
 
 Before Booster, building an event-driven system with the mentioned characteristics required huge investments in hiring engineers with the needed expertise. Booster packs this expertise, acquired from real-case scenarios in high-scale companies, into a very simple tool that handles with the hard parts for you, even provisioning the infrastructure!
 
-We have redesigned the whole developer experience from scratch, taking advantage of the advanced TypeScript type system and Serverless technologies to go from project generation to a production-ready real-time GraphQL API that can ingest thousands of concurrent users in a matter of minutes.
+We have redesigned the whole developer experience from scratch, taking advantage of the advanced TypeScript type system and Serverless technologies to go from project generation to a production-ready application in the cloud composed by a set of orchastrated services which provides a real-time GraphQL API that can ingest thousands of concurrent users in a matter of minutes.
```

Comment:
`application in the cloud composed by a set of orchastrated services which provides` -> `application in the cloud which provides` 

`orhcestrated services` sounds weird, saying we are in the cloud already infers we are using their services.

Reply:
You are right. I will change that! Thank you :D